EDI/Operation/01_ManageDefinitions.md

```diff
@@ -0,0 +1,249 @@
+[!!Exports](../UserInterface/01a_Exports.md)
+
+
+# Manage definitions 
+
+In Actindo, the definition of the EDIFACT export messages specifies the following:
+- Basic settings with template engine and connection to be used
+- File settings with the message structure and related attributes   
+- Data sources with attribute mappings  
+- Mapping settings
+
+Each EDIFACT export message requires two definitions in the *DataHub Exporter* module for each business partner and EDIFACT message:
+- Definition for the message header   
+    Figuratively speaking, the message header is the envelope, with which the message is sent. It contains general data such as receiver/sender information and dates. 
+    
```

Comment:
Schöne bildliche Darstellung :)
Vorschlag: the message header is the envelope in which the message is sent.